api/src/pcapi/core/mails/transactional/pro/invoice_available_to_pro.py

```diff
@@ -50,4 +55,4 @@ def send_invoice_available_to_pro_email(invoice: finance_models.Invoice, batch:
         if not invoice.reimbursementPoint.bookingEmail:
             return
         recipients = [invoice.reimbursementPoint.bookingEmail]
-    mails.send(recipients=recipients, data=data)
+    mails.send(recipients=set(recipients), data=data)
```

Comment:
C'est du détail mais on peut faire `{venue_link.venue.bookingEmail for venue_link in bank_account.venueLinks if venue_link.venue.bookingEmail}` plus haut pour directement avoir un set plutôt de construire la liste pour ensuite de toute façon la caster en set

Reply:
Certes